src/main/java/com/microsoftopentechnologies/windowsazurestorage/helper/AzureStorageAccount.java

```diff
@@ -143,30 +142,30 @@ public AzureCredentials(
      * @deprecated Use {@link #getStorageAccountCredential(Item, String)}.
      */
     @Deprecated
-    public static AzureCredentials.StorageAccountCredential getStorageAccountCredential(String storageCredentialId) {
+    public static AzureStorageAccount.StorageAccountCredential getStorageAccountCredential(String storageCredentialId) {
```

Comment:
Is this any need to keep the inner static class as well? the design seems very strange

Reply:
I am also confused with the inner class here. But I will leave it here temporarily.